src/test/java/core/basesyntax/RegistrationServiceTest.java

```diff
@@ -0,0 +1,113 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceTest {
+    private RegistrationService registrationService;
+
+    @BeforeEach
+    public void setUp() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @Test
+    public void register_ValidUser_Success() {
+        User user = new User("user123", "password123", 18);
+        User registeredUser = registrationService.register(user);
+        assertNotNull(registeredUser.getId());
+    }
+
+    @Test
+    public void register_UserWithShortLogin_ThrowsException() {
+        User user = new User("u", "password123", 18);
+        assertThrows(RegistrationException.class, () -> registrationService.register(user));
```

Comment:
![image](https://github.com/mate-academy/jv-registration-validation/assets/20326858/5b56b528-37eb-4f6d-a8dd-6669d946cd8c)

Reply:
not fixed